### ISSUE TITLE
Error getting bandwidth from Jellyfin

### ISSUE THREAD

**Issue by jonhnjacobjingle** [itschasa/speedrr#7]
**Describe the bug**
First of all, thank you for developing and sharing this.
When playing several different Jellyfin H264 mp4/mkv streams (7-15 Mbps range), the following error was encountered:
```
[2024-09-26 18:46:35,666] [ERROR] <jellyfin|http://jellyfin:8096> Error getting bandwidth:
Traceback (most recent call last):
  File "/home/modules/media_server.py", line 145, in run
    bandwidth = int(self.get_bandwidth() * self._server_config.bandwidth_multiplier)
  File "/home/modules/media_server.py", line 257, in get_bandwidth
    bandwidth += int(stream["BitRate"])
KeyError: 'BitRate'
 (media_server.py:147)
```

**To Reproduce**
1. Set up speedrr
2. Stream from Jellyfin (but oddly not any stream, see additional context below)

**Additional context**
This error is not encountered with all streams. When playing a low bitrate VP9 mp4 stream, the qBittorrent upload speed was adjusted as expected:
```
[2024-09-26 18:42:45,088] [INFO] Update event triggered (main.py:77)
[2024-09-26 18:42:45,088] [INFO] <media_servers> Reduction values = http://jellyfin:8096: 0 (media_server.py:44)
[2024-09-26 18:42:45,088] [INFO] <schedule> Reduction values = 00:00-23:59: 0.0 (schedule.py:25)
[2024-09-26 18:42:45,088] [INFO] New calculated upload speed: 22.0mbit (main.py:85)
[2024-09-26 18:42:45,089] [INFO] Getting active torrent counts (main.py:87)
[2024-09-26 18:42:45,748] [INFO] Set upload speed for http://qbittorrent:8080 to 22.0mbit (main.py:108)
[2024-09-26 18:42:45,748] [INFO] Upload speeds updated (main.py:111)
[2024-09-26 18:42:45,748] [INFO] Waiting for next update event (main.py:118)
[2024-09-26 18:45:35,556] [INFO] Update event triggered (main.py:77)
[2024-09-26 18:45:35,557] [INFO] <media_servers> Reduction values = http://jellyfin:8096: 1.113 (media_server.py:44)
[2024-09-26 18:45:35,557] [INFO] <schedule> Reduction values = 00:00-23:59: 0.0 (schedule.py:25)
[2024-09-26 18:45:35,557] [INFO] New calculated upload speed: 20.887mbit (main.py:85)
[2024-09-26 18:45:35,557] [INFO] Getting active torrent counts (main.py:87)
[2024-09-26 18:45:36,236] [INFO] Set upload speed for http://qbittorrent:8080 to 20.887mbit (main.py:108)
[2024-09-26 18:45:36,236] [INFO] Upload speeds updated (main.py:111)
[2024-09-26 18:45:36,236] [INFO] Waiting for next update event (main.py:118)
```
My speedrr config.yaml:
```
units: mbit
min_upload: 1
max_upload: 22
clients:
  - type: qbittorrent
    url: http://qbittorrent:8080
    username:
    password:
    https_verify: true
modules:
  media_servers:
    - type: jellyfin
      url: http://jellyfin:8096
      api_key:
      https_verify: true
      bandwidth_multiplier: 1
      update_interval: 5
      ignore_local_streams: true
      ignore_paused_after: 300
  schedule:
    - start: "00:00"
      end: "23:59"
      days: [all]
      upload: 1%
```

**Comment by itschasa** [comment]
Should have been fixed with #6
I've made a new release for it now (v0.0.2), can you try and see if that fixes the issue?

**Comment by jonhnjacobjingle** [comment]
Seems to be fixed! Thank you.
